src/clj/fluree/db/json_ld/commit_data.cljc

```diff
@@ -557,15 +563,18 @@
                              (issuer-flakes db t commit-sid issuer-iri))
         message-flakes     (when message
                              (message-flakes t commit-sid message))
+        txn-flake          (when txn
```

Comment:
I think shadowing function names with local let bindings can be a source of both bugs and confusion. Since there's already a `txn-flake` function definition in scope here could we either change the name of that function to something like `create-txn-flake`, or change the binding here to `txn-f` or something?

Reply:
Thanks for noticing... corrected in 653f8c9c64ed88b0348de53e211d7505a403830a